src/Common/tests/System/IO/FileCleanupTestBase.cs

```diff
@@ -62,6 +62,19 @@ protected string GetTestFilePath(int? index = null, [CallerMemberName] string me
             return Path.Combine(TestDirectory, GetTestFileName(index, memberName, lineNumber));
         }
 
+        /// <summary>
+        /// Creates a non-empty file
+        /// </summary>
+        /// <param name="path">Path to create the file at</param>
+        protected void CreateNonEmptyFile(string path, int fileSize)
```

Comment:
Can you use https://github.com/mjrousos/corefx/blob/UnixGetAttributesFix/src/Common/tests/System/IO/TempFile.cs instead of adding this?